core/steps/network/interface_create.go

```diff
@@ -88,10 +88,6 @@ func (s *createInterface) Do(ctx context.Context) ([]planner.Procedure, error) {
 		return nil, fmt.Errorf("checking if networking interface exists: %w", err)
 	}
 	if exists {
-		// This whole block is unreachable right now, because
-		// the Do function is called only if ShouldDo returns
-		// true. It retruns false if IfaceExists returns true.
-		// Line 76 will never return exists=true
```

Comment:
It's still a dead code here and I left this comment it here:
1. It's visible we try to do that and in case we call Do without ShouldDo, it works
2. If the device status is not update and we someone checks the code "but hey we do an update here", they can see "nope we don't" and don't start debugging for hours to get to the same results that it's not executed if it's called only on ShouldDo==true